src/Datadog.Trace/HttpOverStreams/DatadogHttpClient.cs

```diff
@@ -62,23 +63,22 @@ private HttpResponse ReadResponse(Stream responseStream)
             var stringBuilder = new StringBuilder();
 
             var chArray = new byte[1];
-            void GoNextChar()
+
+            async Task GoNextChar()
             {
-                streamPosition++;
-                chArray[0] = (byte)responseStream.ReadByte();
+                await responseStream.ReadAsync(chArray, offset: 0, count: 1);
```

Comment:
`ConfigureAwait(false)`? Here and every other new `await`.

Reply:
I'm too used to writing aspnetcore code where that doesn't matter any more 😳

Reply:
fixed in https://github.com/DataDog/dd-trace-dotnet/pull/1218/commits/1db429214309b593581ad47b1bffbe8b67d599ce